### ISSUE TITLE
More tutorials

### ISSUE THREAD

**Issue by JaGeo** [materialsproject/atomate2#591]
@naik-aakash and I have build some new tutorials for an up-coming talk of mine. It might be interesting to also include such tutorials in the atomate2 documentation.

We have uploaded them here for now:
https://jageo.github.io/TutorialAtomate2Forcefields/intro.html

https://jageo.github.io/sites/Phonon_wf_tutorial_update.html

Would be great to hear what others think about such tutorials and how to integrate them best.

**Comment by JaGeo** [comment]
Might also be related to https://github.com/materialsproject/atomate2/issues/405.

Maybe you, @Andrew-S-Rosen, have some thoughts

**Comment by Andrew-S-Rosen** [comment]
Thanks, @JaGeo! This will be very helpful for @prathami11 😄

**Comment by JaGeo** [comment]
Just fixed one embarrassing mistake in my tutorial. Now, it should hopefully be very useful. 😃

**Comment by Zhuoying** [comment]
@JaGeo Thanks for the tutorial! 
I strongly agree that providing an ipynb is the best way for hands-on tutorials, and if we could have a place to organize all submission tutorials for all the workflows, that would be great!
The Jupyter book you demoed here is an excellent choice (it is also a good idea to show the flow diagram in the notebook). 
@utf @janosh Any thoughts on whether/where we should put all tutorials in?

**Comment by janosh** [comment]
I think the `atomate2` repo is the obvious place to collect such tutorial notebooks. It always bugged me that it took me close to two years from starting to contribute to MP to discover all the examples in https://github.com/materialsvirtuallab/matgenb. This would not have happened if they were in `pymatgen` (most of them show off `pymatgen` functionality).

**Comment by Andrew-S-Rosen** [comment]
I am learning about this right now...!

**Comment by JaGeo** [comment]
When I started contributing to them, they were in pymatgen, I think. (@Andrew-S-Rosen : 🙈). This also lead to all kinds of issues. Keeping the updated wasn't easy, I think..

Would be great to collect them in this repo here. We just need to update them frequently.

**Comment by utf** [comment]
I think it would be great to have tutorials in the atomate2 repo and on the docs. In terms of keeping them up to date, I wonder if there is any way we can have some sort of automated testing for them. This might work for the forcefield ones, but will be more difficult for the DFT ones.

**Comment by JaGeo** [comment]
Could one use the mock_* methods within the tutorials without them showing up?

**Comment by janosh** [comment]
@JaGeo Yes, Jupyter supports cell metadata which can contain tags. E.g.

```json
{
    "jupyter": {
        "outputs_hidden": true
    },
    "tags": [
        "remove-input",
    ]
}
```

will hide both input and output of a notebook cell when converted to docs. This does not affect the notebook itself, just it's HTML conversion. Afaik this works with `nbconvert`, Sphinx and MyST-NB.

**Comment by ml-evs** [comment]
I'm just using the new `MPGGARelax` (etc) workflows for the first time, happy to contribute an example use case taking a new material and screening it against MP results; guess this wouldn't be a direct tutorial though. 

More generally I don't really follow the current structure of the online docs (e.g., the 'chaining workflows' section being hidden under Codes -> VASP) -- happy to do a pass on this if it isn't already in progress.
